gammapy/scripts/fermi_3FGL_info.py

```diff
@@ -0,0 +1,32 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from __future__ import print_function, division
+import logging
+logging.basicConfig(level=logging.INFO, format='%(levelname)s - %(message)s')
+import os
+from astropy.units import Quantity
+from astropy.io import fits
+from ..utils.scripts import get_parser
+
+__all__ = ['fermi_3FGL_info']
+
+
+def print_info(source_name):
```

Comment:
Can you make this a `Fermi3FGLObject.info` method?

I like the idea to have a command line tool to query Fermi catalogs and make plots (see #203).
It's also good to start small, e.g. even just printing info from the 3FGL catalog and a test that this works.

I haven't written up docs on how to add command line utils, but in a nutshell you should take a working example as a template, e.g.:
https://github.com/gammapy/gammapy/blob/master/gammapy/scripts/bin_image.py
https://github.com/gammapy/gammapy/blob/master/gammapy/scripts/tests/test_bin_image.py

And you'll have to add the tool to this list:
https://github.com/gammapy/gammapy/blob/master/setup.py#L81